lib/chef/knife/ec_base.rb

```diff
@@ -120,110 +128,128 @@ def org_admin
         rescue Net::HTTPServerException => ex
           knife_ec_error_handler.add(ex)
         end
-      end
+  end
 
       def server
         @server ||= if Chef::Config.chef_server_root.nil?
-                      ui.warn("chef_server_root not found in knife configuration; using chef_server_url")
+                      ui.warn('chef_server_root not found in knife configuration; using chef_server_url')
                       Chef::Server.from_chef_server_url(Chef::Config.chef_server_url)
                     else
                       Chef::Server.new(Chef::Config.chef_server_root)
-                    end
       end
+        end
 
       # Since knife-ec-backup hasn't been updated to use API V1 keys endpoints
       # we should explicitly as for V0.
       def rest
-        @rest ||= Chef::ServerAPI.new(server.root_url, {:api_version => "0"})
-      end
+        @rest ||= Chef::ServerAPI.new(server.root_url, api_version: '0')
+        end
 
       def remote_users
         @remote_users ||= rest.get('/users')
-      end
+        end
 
       def remote_user_list
         @remote_user_list ||= remote_users.keys
-      end
+        end
 
       def local_user_list
         @local_user_list ||= Dir.glob("#{dest_dir}/users/*\.json").map { |u| File.basename(u, '.json') }
-      end
+        end
 
       def users_for_purge
         # not itended to be called from ec_base
         raise Chef::Knife::EcBase::UnImplemented
-      end
+        end
 
       def knife_ec_error_handler
         @knife_ec_error_handler ||= Chef::Knife::EcErrorHandler.new(dest_dir, self.class)
-      end
+        end
 
       def user_acl_rest
         @user_acl_rest ||= if config[:skip_version]
                              rest
                            elsif server.supports_user_acls?
                              rest
                            elsif server.direct_account_access?
-                             Chef::ServerAPI.new("http://127.0.0.1:9465", {:api_version => "0"})
-                           end
-
+                             Chef::ServerAPI.new('http://127.0.0.1:9465', api_version: '0')
       end
+        end
 
       def set_skip_user_acl!
         config[:skip_useracl] ||= !(server.supports_user_acls? || server.direct_account_access?)
-      end
+        end
 
       def set_client_config!
-        Chef::Config.custom_http_headers = (Chef::Config.custom_http_headers || {}).merge({'x-ops-request-source' => 'web'})
+        Chef::Config.custom_http_headers = (Chef::Config.custom_http_headers || {}).merge('x-ops-request-source' => 'web')
         Chef::Config.node_name = 'pivotal'
         Chef::Config.client_key = webui_key
-      end
+        end
```

Comment:
There are a whole bunch of these two spaces added before `end` so that the `end` doesn't line up with the function definition.